### PR TITLE
make jrcs external dependency

### PR DESCRIPTION
**Pull request by vladak**
Replaces the `jrcs` module with dependencies. The annotation is performed by parsing the output from the `blame` command.

### CODE REVIEW

File: opengrok-indexer/src/main/java/org/opengrok/indexer/history/RCSRepository.java [hunk 1]
```diff
@@ -79,37 +86,23 @@ boolean fileHasAnnotation(File file) {
 
     @Override
     Annotation annotate(File file, String revision) throws IOException {
-        File rcsFile = getRCSFile(file);
-        return rcsFile == null ? null : annotate(file, revision, rcsFile);
-    }
+        ArrayList<String> argv = new ArrayList<>();
```
Comment (Orviss) [inline opengrok-indexer/src/main/java/org/opengrok/indexer/history/RCSRepository.java]: Maybe use `List` interface?

File: opengrok-indexer/src/main/java/org/opengrok/indexer/history/RCSRepository.java [hunk 2]
```diff
@@ -47,6 +44,16 @@
 
     private static final long serialVersionUID = 1L;
 
+    /**
+     * the property name used to obtain the command to get annotation for this repository
```
Comment (Orviss) [inline opengrok-indexer/src/main/java/org/opengrok/indexer/history/RCSRepository.java]: Javadoc should have sentences - first sentence will be brief info. So I would start with upper case letter and end with `.`. CheckStyle has some checks for this. We can enable them later if the need arises.

File: opengrok-indexer/src/main/java/org/opengrok/indexer/history/RazorRepository.java [hunk 3]
```diff
@@ -256,10 +256,6 @@ InputStream getHistoryGet(String parent, String basename, String rev) {
     Annotation annotate(File file, String revision)
             throws IOException {
         // @TODO : Rename & Delete Support
-        File rcsFile = getRazorArchiveRCSFileFor(file);
-        if (rcsFile != null && rcsFile.exists()) {
-            return RCSRepository.annotate(file, revision, rcsFile);
-        }
```
Comment (tulinkry) [inline opengrok-indexer/src/main/java/org/opengrok/indexer/history/RazorRepository.java]: why is this no longer needed?

Comment (vladak) [reply opengrok-indexer/src/main/java/org/opengrok/indexer/history/RazorRepository.java]: It should work as long as Razor automatically creates the `RCS/XYZ.txt,v` file.

**Pull request merged**